cfgov/v1/wagtail_hooks.py

```diff
@@ -365,21 +366,49 @@ def hide_snippets_menu_item(request, menu_items):
                      if item.url != reverse('wagtailsnippets:index')]
 
 
-# Override list of allowed tags in a RichTextField
-@hooks.register('construct_whitelister_element_rules')
-def whitelister_element_rules():
-    allow_html_class = attribute_rule({
-        'class': True,
-        'itemprop': True,
-        'itemscope': True,
-        'itemtype': True,
-    })
-
-    allowed_tags = ['aside', 'h4', 'h3', 'p', 'span',
-                    'table', 'tr', 'th', 'td', 'tbody', 'thead', 'tfoot',
-                    'col', 'colgroup']
-
-    return {tag: allow_html_class for tag in allowed_tags}
+if wagtail.VERSION < (2, 0):
+    # Override list of allowed tags in a RichTextField
+    @hooks.register('construct_whitelister_element_rules')
+    def whitelister_element_rules():
+        allow_html_class = attribute_rule({
+            'class': True,
+            'itemprop': True,
+            'itemscope': True,
+            'itemtype': True,
+        })
+
+        allowed_tags = ['aside', 'h4', 'h3', 'p', 'span',
+                        'table', 'tr', 'th', 'td', 'tbody', 'thead', 'tfoot',
+                        'col', 'colgroup']
+
+        return {tag: allow_html_class for tag in allowed_tags}
+
+
+# construct_whitelister_element_rules are depricated in Wagtail 2.0
+# https://docs.wagtail.io/en/stable/releases/
+# 2.0.html#construct-whitelister-element-rules-hook-is-deprecated
+if wagtail.VERSION >= (2, 0):
```

Comment:
Let's make this an `elif` so it doesn't always check twice, and make the comment more specific.

```suggestion
elif wagtail.VERSION >= (2, 0):
    # The construct_whitelister_element_rules hook was deprecated in Wagtail 2,
    # so we'll use register_rich_text_features instead.
    # Only applies to Hallo editors, which only remain in our custom
    # AtomicTableBlock table cells.
```